tests/test_urlslug.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+# Copyright (c) 2019, Rice University
+# This software is subject to the provisions of the GNU Affero General
+# Public License version 3 (AGPLv3).
+# See LICENCE.txt for details.
+"""
+import re
+
+import pytest
+
+from cnxcommon.urlslug import generate_slug
+
+def test_generate_slug():
```

Comment:
do the page titles in the db contain html entities or are those encoded when the archive response is put together? in rex we had to decode html entities _before_ stripping out special characters, but i'm not sure if that will be a concern for you guys.

Reply:
Yeah, I'm also working with what _you_ had available. I've updated the code.

`title = re.sub(r"<.*?>", "", title)  # remove html tags`